addons/binding/org.openhab.binding.network.test/pom.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
`mvn clean install -B -V`

Unfortunately just test does not execute the tests, this is because of the osgi maven test plugin we use 🙁